rules/windows/process_creation/proc_creation_win_sharp_impersonation_tool.yml

```
@@ -0,0 +1,38 @@
+title: SharpImpersonation Execution
+id: f89b08d0-77ad-4728-817b-9b16c5a69c7a
+related:
+    - id: cf0c254b-22f1-4b2b-8221-e137b3c0af94
+      type: similar
+status: experimental
+description: Detects execution of the SharpImpersonation tool. Which can be used to manipulate tokens on a Windows computers remotely (PsExec/WmiExec) or interactively
+references:
+    - https://s3cur3th1ssh1t.github.io/SharpImpersonation-Introduction/
+    - https://github.com/S3cur3Th1sSh1t/SharpImpersonation
+author: Sai Prashanth Pulisetti @pulisettis, Nasreddine Bencherchali
+date: 2022/12/27
+tags:
+    - attack.privilege_escalation
+    - attack.defense_evasion
+    - attack.t1134.001
+    - attack.t1134.003
+logsource:
+    product: windows
+    category: process_creation
+detection:
+    selection_img:
+        - Image|endswith: '\SharpImpersonation.exe'
+        - OriginalFileName: 'SharpImpersonation.exe'
+    selection_cli:
+        - CommandLine|contains|all:
+            - '.exe user:'
+            - ' binary:'
+        - CommandLine|contains|all:
+            - '.exe user:'
```

```suggestion
            - ' user:'
```
no order in the param [ArgParse.cs](https://github.com/S3cur3Th1sSh1t/SharpImpersonation/blob/c255ca75ac05e6f0bd7f21dba44e2a09a34461ee/SharpImpersonation/ArgParse.cs#L11)